src/js/base/moment.js

```diff
@@ -45,7 +45,7 @@ Handlebars.registerHelper({
   formatMoment(date, format, { hash = {} }) {
     if (!date) return new Handlebars.SafeString(hash.defaultHtml || '');
 
-    date = moment(date, hash.inputFormat);
+    date = hash.utc ? moment.utc(date, hash.inputFormat) : moment(date, hash.inputFormat);
```

Comment:
I think I was incorrect.. as below I _believe_ this'll need `moment.utc(...).local()` so that the date is read as utc but stored locally as it would be if it included the utc indicator in the string.